students/chankyu/users/views.py

```diff
@@ -35,5 +35,21 @@ def post(self, request):
             )
             return HttpResponse({"MESSAGE" : "SUCCESS"}, status = 201)
 
+        except:
+            return HttpResponse({"MESSAGE" : "KEY_ERROR"}, status = 400)
+
+class SigninView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email    = data['email']
+            password = data['password']
+
+            if not User.objects.filter(email = email, password = password).exists():
+                return HttpResponse({"MESSAGE" : "INVALID_USER"}, status = 401)
+
+            return HttpResponse({"MESSAGE": "SUCCESS"}, status = 200)
+
         except:
```

Comment:
아래와 같이 KeyError를 추가해주세요!

```suggestion
        except KeyError:
```

Reply:
놓쳤습니다. 번거롭게 해드려서 죄송합니다.  반영하였습니다.

Reply:
❤️